components/org.wso2.carbon.identity.api.server.claim.management/org.wso2.carbon.identity.rest.api.server.claim.management.v1/src/main/java/org/wso2/carbon/identity/rest/api/server/claim/management/v1/core/ServerClaimManagementService.java

```diff
@@ -1531,4 +1544,57 @@ private void validateClaimModificationEligibility() throws ClaimMetadataClientEx
         }
 
     }
+
+    private boolean isSubOrganizationContext() throws ClaimMetadataClientException {
+
+        try {
+            String organizationId =
+                    getOrganizationManager().resolveOrganizationId(ContextLoader.getTenantDomainFromContext());
+            return !getOrganizationManager().isPrimaryOrganization(organizationId);
```

Comment:
OrganizationUtils.isOrganization(ContextLoader.getTenantDomainFromContext())

could be easily used for this case

Reply:
Updated with [7bcd42b](https://github.com/wso2/identity-api-server/pull/771/commits/7bcd42b531f32433a8253deb53cd32e381ad7d5a)